contract/src/msgs/sorted_set.rs

```diff
@@ -0,0 +1,99 @@
+use seda_common::msgs::data_requests::DataRequest;
+
+use super::*;
+
+pub type IndexKey = (u128, u64, Hash);
```

Comment:
Same here... we should explain what each element of the "tripple" it is:
- price
- height
- dr_id